.github/workflows/ci.yml

```diff
@@ -162,7 +162,7 @@ jobs:
       - cancel-previous-runs
       - lint-toml-files
       - cargo-verifications
-      - publish-crates
+    # - publish-crates
```

Comment:
Why is this commented out?

Reply:
Agreed, we should not be disabling our entire publishing pipeline from master for the sake of debugging this deployment.

Reply:
This is only a dependency, which actually isn't even being respected, if you see 2 lines below we're doing `if: always()`

I actually meant to completely remove it